mac.js

```diff
@@ -12,7 +12,7 @@ class MacApp {
   constructor (opts, stagingPath) {
     this.opts = opts
     this.stagingPath = stagingPath
-    this.appName = opts.name
+    this.appName = common.sanitizeAppName(opts.name)
```

Comment:
I think this change may be too heavy-handed - I believe it changes the name shown in the OSX metadata too. I'll have to write a (failing?) test for this.

This is the main reason why I tagged this PR as `[WIP]`.